src/Symfony/Component/Yaml/Yaml.php

```diff
@@ -29,6 +29,7 @@ class Yaml
     const DUMP_OBJECT_AS_MAP = 64;
     const DUMP_MULTI_LINE_LITERAL_BLOCK = 128;
     const PARSE_CONSTANT = 256;
+    const PARSE_CUSTOM_TAGS = 512;
```

Comment:
I wonder if people won't think that built-in tags are also converted to a `TaggedValue` with this name. Wdyt?